XLForm/XL/Cell/XLFormSelectorCell.m

```diff
@@ -274,6 +283,8 @@ -(void)formDescriptorCellDidSelectedWithFormController:(XLFormViewController *)c
                                                                   }]];
             }
             [formViewController presentViewController:alertController animated:YES completion:nil];
+            // Fix for "Snapshotting a view that has not been rendered results in an empty snapshot." bug
+            [[alertController view] layoutIfNeeded];
```

Comment:
I am testing Example app in the simulator and this warning did not appear. Also in your test app it does not (tested iPad Pro). I do not think we should include it.